src/containers/find-offer/offer-container/OfferContainer.tsx

```diff
@@ -29,6 +29,7 @@ const OfferContainer: FC<OfferContainerProps> = ({
   const { t } = useTranslation()
   const { isMobile, isLaptopAndAbove } = useBreakpoints()
   const { setChatInfo } = useChatContext()
+  const { userId } = useAppSelector((state) => state.appMain)
 
   const onBookmarkClick = (id: string) => {
     console.log(id)
```

Comment:
Delete console, please

Reply:
for now this function is mocked, I can remove or make it to return null, but I don`t think it make sense